02.calendar/cal.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+// Main関数
+const main = () => {
+  const options = cutInput(process.argv.slice(2)) // 入力された引数を切り分け
+  console.log('    ', String(options.m), '月  ', String(options.y)) // 一行目の年月の表示
```

Comment:
```suggestion
  console.log(`    ${options.m}月  ${options.y}`)
```

バッククオート文字列を使って変数を埋め込む方がスッキリ書けそうです。

Reply:
バッククオート文字列(テンプレート文字列)で、式展開させる方法をJavaScriptでは取れるのですね。

テンプレート文字列については『スラスラ読める JavaScriptふりがなプログラミング』より「エスケープ文字の改行`\n`ではない改行を入力できるもの」という理解でした。